plugin-server/src/cdp/cdp-consumers.ts

```diff
@@ -422,12 +425,29 @@ abstract class CdpConsumerBase {
                     })
                 })
 
-                const results = (
-                    await this.runManyWithHeartbeat(notOverflowedInvocations, (item) =>
+                const notExceptionInvocations = notOverflowedInvocations.filter((item) => {
```

Comment:
Is this the right step to be doing the split? I don't really know what "overflow invocations". Should the error tracking custom masking (snoozing / archiving) be done before / after / instead of anything here?